db/migrate/20220311191446_add_time_zone_to_storage_locations.rb

```diff
@@ -0,0 +1,7 @@
+class AddTimeZoneToStorageLocations < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      add_column :storage_locations, :time_zone, :string, null: false, default: 'America/Los_Angeles'
```

Comment:
According to strong_migrations, this is no longer unsafe if our Postgres version is >= 11 (which I assume it is).